cpp/src/parquet/properties.h

```diff
@@ -1067,12 +1068,26 @@ class PARQUET_EXPORT ArrowWriterProperties {
       return this;
     }
 
+    /// \brief Set time columns as adjusted to utc (default)
```

Comment:
IMO, "time column" is a weird name, since we have so many "times" (date, time32, time64, timestamp), we'd better  note this is just time32/time64 in our comment